src/components/screens/new-questionnaire-screen.tsx

```diff
@@ -34,30 +35,35 @@ const NewQuestionnaireScreen = () => {
   const { questionnairesApi } = useLambdasApi();
   const [loading, setLoading] = useState(false);
   const setError = useSetAtom(errorAtom);
+  const [tooltipMessage, setTooltipMessage] = useState<string>("");
+  const [isDisabled, setIsDisabled] = useState<boolean>(true);
```

Comment:
Rather than setting this to be true to begin with, why not set it based on the actual values we want to track for disabling e.g. where there is title and description?

Reply:
In fact I think rather than having this isDisabled state you could reference the questionnaire.title || questionnaire.description in the disabled attribute of the button component where it is used.

I think this will save a lot of updating and tracking of the disabled state which maybe isn't necessary/ might be a little overcomplicated. What do you think?